NLog.Windows.Forms.Tests/packages.config

```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NLog" version="4.0.0" targetFramework="net35" />
-  <package id="xunit" version="1.9.2" targetFramework="net35" />
+  <package id="NLog" version="4.2.2" targetFramework="net45" />
```

Comment:
Sorry, it got there when I was fighting with xUnit versions.
Reverting to Nlog 4.0.1. Version 4.0.0 is not able to find the layout renderer.